CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- **File Upload**:
+  - changed error text of uploaded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```

Comment:
Hi @Usmanfee we have an continuous issue in the changelog file. I suggest you revert any change to the changelog and add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.